packages/block-library/src/paragraph/theme.scss

```diff
@@ -0,0 +1,4 @@
+p {
+	margin-top: $default-block-margin;
```

Comment:
This could also be 2em if we thought that was nicer.

Reply:
I'd prefer if we could stick to `em`-based sizes for all text-based measurements in general. It looks like we're already doing that with top/bottom margins for our headings, so we should continue that here.